test/integration/utils/expectations.ts

```diff
@@ -0,0 +1,41 @@
+import { Collection, Key } from '../types';
+
+function getStep(key: Key, times: number = 1): Collection {
+  return { key, times };
+}
+
+const expectations: Collection = {
```

Comment:
If we add more tests this is the object that would contain all the "expectations" for those as well? Not sure that is a great idea, seems like it is likely to become really bloated and hard to read. Would probably prefer if we keep separate files for each test or set of tests. But anyway I think it's fine for now.

Reply:
The original plan was one test file - one expectation file. This one is could be `keyboard.expectations` in `expectations` folder. Other test files could have their own expectation pair.
I think I'm going to rename and move this, but keep the readability of the expectation file in focus